README.md

```diff
@@ -6,4 +6,6 @@ Creating a *fork* is producing a personal copy of someone else's project. Forks
 
 After forking this repository, you can make some changes to the project, and submit [a Pull Request](https://github.com/octocat/Spoon-Knife/pulls) as practice.
 
+PS: Make some changes in your forked branch to be able to submit a valid pull request!
```

Comment:
I think you should make it sound more… professional? (for lack of a better word)